### PR TITLE
feat: Autonomy limit order

### PR DESCRIPTION
**Pull request by legend620**
Implement Limit Order feature in Apeswap Exchange according to this Figma design.

https://www.figma.com/file/RX1z6bNxw8NpRGhNQ1B2Gl/Margin?node-id=0%3A1

@obiedobo , please have a look. Thanks!

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import Menu from './components/Menu'
 import ToastListener from './components/ToastListener'
 import PageLoader from './components/PageLoader'
 import Pool from './views/Pool'
+import Orders from './views/Orders'
```
Comment (Harambe-Nakamoto) [inline src/App.tsx:17]: Could you lazy load Orders below?

File: .eslintrc [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "import/no-extraneous-dependencies": 0,
     "react/require-default-props": 0,
     "react/no-unused-prop-types": 0,
+    "react-hooks/exhaustive-deps": 0,
```
Comment (Harambe-Nakamoto) [inline .eslintrc:34]: Anyway you can resolve these without disabling it on the linter?

File: src/views/Orders/redirects.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        ...location,
+        pathname: '/swap',
+        search:
+          search && search.length > 1
+            ? `${search}&outputCurrency=${outputCurrency}`
+            : `?outputCurrency=${outputCurrency}`,
+      }}
+    />
+  )
+}
```
Comment (Harambe-Nakamoto) [inline src/views/Orders/redirects.tsx:31]: Is this file being used?

File: src/App.tsx (hunk 1 above)
Comment (legend620) [reply src/App.tsx]: Fixed this issue, please check

File: .eslintrc (hunk 2 above)
Comment (legend620) [reply .eslintrc]: Just disabled this rule for one line, please check

**Comment by obiedobo** [comment]
Looks good. Closing to open a native PR.

**Pull request closed without merging**